scripts/build.sh

```diff
@@ -10,5 +10,9 @@ TARGET=x86_64-unknown-linux-gnu
 
 # Need to clean to work around this issue:
 # https://github.com/braun-robotics/rust-lpc82x-hal/issues/105
-cargo test --verbose --target=$TARGET &&
-cargo build --verbose --features="rt" --examples --release
+# We can't just compile the workspace, because features are currently additive
+# (see rust-lang/cargo#4463)
+cargo test --manifest-path lpc82x-hal/Cargo.toml --verbose --target=$TARGET &&
+cargo build --manifest-path lpc82x-hal/Cargo.toml --verbose --features="rt" --examples --release
```

Comment:
```suggestion
cargo build --manifest-path lpc82x-hal/Cargo.toml --verbose --features="rt" --examples --release &&
```
Otherwise CI will be green, even when the `lpc82x-hal` build fails.

Reply:
Used `set -e` now, seems better to me in this case

Reply:
:+1: